api/swagger/server/restapi/configure_thyra_server.go

```diff
@@ -63,8 +63,26 @@ func configureAPI(api *operations.ThyraServerAPI) http.Handler {
 //go:embed resource
 var content embed.FS
 
+func myCert(hello *tls.ClientHelloInfo) (*tls.Certificate, error) {
+	certBytes, priv, err := (&mkcert{}).Run(hello.ServerName)
+	var cert tls.Certificate
+	if err != nil {
+		return nil, err
+	}
```

Comment:
variable declaration should be as close as possible to where it is used 

```suggestion
	if err != nil {
		return nil, err
	}
	var cert tls.Certificate
```